lib/util/exec/index.ts

```diff
@@ -99,10 +97,11 @@ export async function exec(
   const extraEnv = { ...opts.extraEnv, ...customEnvVariables };
   let cwd;
   // istanbul ignore if
+  const { localDir } = getAdminConfig();
   if (cwdFile) {
-    cwd = join(execConfig.localDir, dirname(cwdFile));
+    cwd = join(localDir, dirname(cwdFile));
   }
-  cwd = cwd || opts.cwd || execConfig.localDir;
+  cwd = cwd || opts.cwd || localDir || null;
```

Comment:
Sorry, I'm not familiar with this area of code, but is there a significance with `null` over `undefined`?